src/App.ts

```diff
@@ -166,6 +166,8 @@ export default class App {
 
   private axios: AxiosInstance;
 
+  private installerOptions: ExpressReceiverOptions['installerOptions'];
```

Comment:
since `installerOptions` is just passed through to the `ExpressReceiver`, can we avoid storing it as a property in `App`?

just picked up on the code smell of a property's type being defined in terms of another class's property.

Reply:
The reason why I store it as a property is because I have to add `clientOptions` to `installerOptions`. You can see that at https://github.com/slackapi/bolt-js/pull/586/files#diff-0286657a0789ea9446fa3de979ff3e09R226-R229.